### PR TITLE
fix: Hamburg date

### PR DESCRIPTION
**Pull request by ragnarok22**

### CODE REVIEW

File: src/data/future-events.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 - city: Hamburg
-  date: '2020-12-31'
+  date: '2021-12-11'
   where: Online
```
Comment (marcoacierno) [inline src/data/future-events.yaml:3]: I think we should change this as well?

```suggestion
  where: "Gasstraße 6A, 22761 Hamburg"
```

**Comment by marcoacierno** [comment]
Do we want to change the hero as well?

image

**Comment by ragnarok22** [comment]
> 

This commit change the hero
image

**Comment by marcoacierno** [comment]
Ah right 🤦 I thought it only updated the cards below


I am not sure why this website doesn't have a PR preview 🤔

**Review by marcoacierno — APPROVED** [review]

**Pull request merged**